tools/development/parser/meson.build

```diff
@@ -4,7 +4,7 @@ flex_cdata = configuration_data()
 flex_min_version='2.5.31'
 flex = find_program('flex', 'win_flex')
 
-flexversion_res = run_command([flex, '--version'])
+flexversion_res = run_command([flex, '--version'], check : true)
```

Comment:
I think you just want to use `find_program('flex', 'win_flex', version: '>=2.5.31')` here and skip all this manual version checking?

Reply:
Thanks! It is applied.